.gitignore

```diff
@@ -38,6 +38,7 @@ package-lock.json
 
 # [build] Built files
 /build/
+/dist/
```

Comment:
you need to move this to another block,
every `[bloc]` is ignore files generated by the run generator. If you think this line is generic to each project you should do a PR to the generator, otherwise you can put it at the bottom of the `.gitignore`.